externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/KyuubiSparkUtil.scala

```diff
@@ -62,9 +66,9 @@ object KyuubiSparkUtil extends Logging {
     // scalastyle:off line.size.limit
     // format: off
     s"""
-       |           Spark application name: ${sc.appName}
+       |           Spark application name: $engineName
        |                 application ID: $engineId
-       |                 application web UI: $webUrl
```

Comment:
The webUrl(L63) is not used and needs to be removed.

Reply:
resolved, thanks